compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/NodeEntry.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Could you please update the year with this PR ? This came from my original PR.

Reply:
Done